app/assets/javascripts/application.js

```diff
@@ -23,3 +23,19 @@ $(window).scroll(function() {
 		$('.content').removeClass('fix-content');
   }
 });
+
+// Smooth Scroll //
```

Comment:
Can you upgrade application.js file to application.coffee please ? (to know coffeescript in your skills)